### ISSUE TITLE
Tutorial file doesn't exist: `moveit_ros/moveit_servo/launch/servo_example.launch.py`

### ISSUE THREAD

**Issue by VladimirFokow** [moveit/moveit2_tutorials#905]
### Description

[This tutorial](https://moveit.picknik.ai/main/doc/how_to_guides/controller_teleoperation/controller_teleoperation.html#launch-file) says:

> The file that launches this example is 
> `ws_moveit2/src/moveit2/moveit_ros/moveit_servo/launch/servo_example.launch.py`

But this file doesn't exist on [main](https://github.com/ros-planning/moveit2/tree/main/moveit_ros/moveit_servo/launch) or iron branches. 

<br />

For information:

it exists on the [humble](https://github.com/ros-planning/moveit2/tree/humble/moveit_ros/moveit_servo/launch) branch.
It was described also in [this humble tutorial](https://moveit.picknik.ai/humble/doc/examples/realtime_servo/realtime_servo_tutorial.html)  (in rolling docs this same page is now rewritten and contains some different info).

<br />

The file was refactored in [this PR](https://github.com/ros-planning/moveit2/pull/2224/files#diff-c2ef5a7cd70397ce7d6b018fb7aef8947923fabac681c7897b08cc577d033983): renamed to `demo_ros_api.launch.py` + other changes were done to it.

The tutorial was written [here](https://github.com/ros-planning/moveit2_tutorials/pull/461) by @henrygerardmoore

...P.S. Probably this issue should be in https://github.com/ros-planning/moveit2_tutorials ? 

### Your environment
* ROS Distro: Humble
* OS Version: Ubuntu 22.04
* Binary build
* binary release version: jammy-humble
* RMW: Cyclone DDS

### Expected behaviour
Docs should point to existing files, 
and the commands should be correct.

### Actual behaviour
File doesn't exist, 
some commands use the old names - can't be executed.

**Comment by sea-bass** [comment]
Yep, seems the MoveIt Servo refactor from last year on `main` did away with this example.

You can find it on the `humble` branch of the `moveit2` repo, though: https://github.com/moveit/moveit2/blob/humble/moveit_ros/moveit_servo/launch/servo_example.launch.py

If you are working on this and are able to port this example to the new MoveIt Servo, that would be great.
